Datadog/Example/ExampleAppDelegate.swift

```diff
@@ -81,12 +78,15 @@ class ExampleAppDelegate: UIResponder, UIApplicationDelegate {
         #endif
 
         // Register Tracer
-        Global.sharedTracer = Tracer.initialize(
-            configuration: Tracer.Configuration(
-                sendNetworkInfo: true
+        DatadogTracer.initialize(
+            configuration: DatadogTracer.Configuration(
+                sendNetworkInfo: true,
+                customIntakeURL: Environment.readCustomTraceURL()
             )
         )
 
+        Global.sharedTracer = DatadogTracer.shared()
+
```

Comment:
I'm wondering if this could be still:

```suggestion
        Global.sharedTracer = DatadogTracer.initialize(
            configuration: DatadogTracer.Configuration(
                sendNetworkInfo: true,
                customIntakeURL: Environment.readCustomTraceURL()
            )
        )
```

It's harder to put some lines in between when it's like this.